client/src/app/containers/Game/models.ts

```diff
@@ -0,0 +1,87 @@
+import { User } from '../../../auth/slice';
+
+export interface Player extends User {
+    color: string;
+}
+
+type Suit = 'Spades' | 'Clubs' | 'Diamonds' | 'Hearts';
+type SuitLetter = 'c' | 'd' | 'h' | 's';
+type ValueLetter =
+    | 'a'
+    | '2'
+    | '3'
+    | '4'
+    | '5'
+    | '6'
+    | '7'
+    | '8'
+    | '9'
+    | '10'
+    | 'j'
+    | 'q'
+    | 'k';
+
+type Value = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13;
+
+// prettier doesn't know about this typescript syntax so we have to disable it here
+// eslint-disable-next-line prettier/prettier
+export type CardName = `${ValueLetter}${SuitLetter}`
+
+export interface Card {
+    name: CardName | 'unknown';
+    suit: Suit;
+    value: Value;
+}
+
+export interface PeggedCard {
+    card: Card;
+    player: string;
+}
+
+export interface Team {
+    players: Player[];
+    color: string;
+    current_score: number;
+    lag_score: number;
+}
+
+export interface Game {
+    id: number;
```

Comment:
:hammer: just because it's usually a number, doesn't mean it'll always be. Really, it should be an opaque string :badpokerface:

Reply:
_at the moment_ :tm:  it's an int on the backend; this highlights the need to decouple the backend's data shape from the frontend's (with an api client layer or network layer or whatever you want to call it), but I'd like to defer that to a follow-up